src/main/java/org/reactome/release/common/CustomLoggable.java

```diff
@@ -0,0 +1,128 @@
+package org.reactome.release.common;
+
+import java.io.Serializable;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.Appender;
+import org.apache.logging.log4j.core.Filter;
+import org.apache.logging.log4j.core.Layout;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.appender.FileAppender;
+import org.apache.logging.log4j.core.appender.RollingRandomAccessFileAppender;
+import org.apache.logging.log4j.core.appender.rolling.RollingRandomAccessFileManager;
+import org.apache.logging.log4j.core.appender.rolling.RolloverStrategy;
+import org.apache.logging.log4j.core.appender.rolling.TriggeringPolicy;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+
+/**
+ * An interface for classes that may want to customise they way they log.
+ * @author sshorser
+ *
+ */
+public interface CustomLoggable
+{
+	/**
+	 * Creates the logger. 
+	 * @param logFileName - custom name for log file.
+	 * @param oldAppenderName - the name of the appender to copy config from.
+	 * @param newAppenderName - the name of the new appender.
+	 * @param append - append, or not.
```

Comment:
The alternative is truncate?

Reply:
I believe so.

Reply:
It might be good to state that explicitly in JavaDoc comment

Reply:
Does this still need to be addressed?